packages/jupyter-chat/style/chat.css

```diff
@@ -194,7 +194,7 @@
 }
 
 .jp-chat-navigation-bottom {
-  bottom: 100px;
+  bottom: 120px;
```

Comment:
We should find a way to position this bottom navigation relatively to the input toolbar, instead of having a fixed size.